src/deepsparse/loggers/async_logger.py

```diff
@@ -59,6 +60,12 @@ def log(self, identifier: str, value: Any, category: MetricCategories):
         )
         job_future.add_done_callback(_log_async_job_exception)
 
+    def __str__(self, level=0):
+        wrapper = textwrap.TextWrapper(initial_indent="\n" + "  " * level)
+        text = wrapper.fill(f"{self.__class__.__name__}:")
```

Comment:
Could we use `__qualname__` instead?
More info here: https://www.codingem.com/qualname-in-python/

Reply:
only class name is relevant to what we're looking for for now, users can dig in for more if needed